include/zephyr/drivers/sensor/grow_r502a.h

```diff
@@ -24,16 +114,76 @@ enum sensor_trigger_type_grow_r502a {
 };
 
 enum sensor_attribute_grow_r502a {
-	/** Add values to the sensor which are having record storage facility */
-	SENSOR_ATTR_R502A_RECORD_ADD = SENSOR_ATTR_PRIV_START,
+	/** To capture finger data and store in RAM buffers*/
+	SENSOR_ATTR_R502A_CAPTURE = SENSOR_ATTR_PRIV_START,
+	/** To create template from RAM buffers during capture*/
+	SENSOR_ATTR_R502A_TEMPLATE_CREATE,
+	/** Add template to the sensor record storage */
+	/*
+	 * @param val->val1	value to store/ add the record
+	 *			to the sensor device's flash
+	 *			library.
+	 */
+	SENSOR_ATTR_R502A_RECORD_ADD,
 	/** To find requested data in record storage */
+	/*
+	 * @result val->val1	value holds finger index matched
+	 *			from sensor's library
+	 *	   val->val2	value holds the matching score
+	 */
 	SENSOR_ATTR_R502A_RECORD_FIND,
 	/** To delete mentioned data from record storage */
+	/*
+	 * @param val->val1	index value where the record
+	 *			stored in sensor device's
+	 *			flash library needs to be
+	 *			deleted.
+	 * @param val->val2	count value till the record
+	 *			needs to be deleted from
+	 *			flash starting from val->val1.
+	 */
 	SENSOR_ATTR_R502A_RECORD_DEL,
-	/** To get available position to store data on record storage */
-	SENSOR_ATTR_R502A_RECORD_FREE_IDX,
 	/** To empty the storage record*/
 	SENSOR_ATTR_R502A_RECORD_EMPTY,
+	/** To get available position to store data on record storage */
+	SENSOR_ATTR_R502A_RECORD_FREE_IDX,
+	/** To load template from storage to RAM buffer of sensor*/
+	/*
+	 * @param val->val1	sensor device's RAM buffer idx
+	 *			values can either be 1 or 2,
+	 *			all other values consider as 2.
+	 * @param val->val2	index value to load the record
+	 *			from sensor device's flash to
+	 *			RAM buffer[val->val1]
+	 */
+	SENSOR_ATTR_R502A_RECORD_LOAD,
+	/** To template data stored in sensor's RAM buffer*/
+	SENSOR_ATTR_R502A_COMPARE,
+	/** To control device LED */
```

Comment:
add comments
"recommended to use R502A_LED_PARAMS in val1"

Reply:
address this pls

Reply:
Already put the comments at line number: 165, should I change the comments same as you commented?

Reply:
got it..